src/coreclr/tools/Common/Compiler/DisplayNameHelpers.cs

```diff
@@ -38,6 +39,12 @@ public static string GetDisplayName(this MethodDesc method)
             {
                 sb.Append(method.OwningType.GetDisplayNameWithoutNamespace());
             }
+            else if (method.GetPropertyForAccessor() is PropertyPseudoDesc property)
+            {
+                sb.Append(property.Name);
+                sb.Append('.');
+                sb.Append(property.GetMethod.Name == method.Name ? "get" : "set");
```

Comment:
I created https://github.com/dotnet/linker/issues/2816 to add this to linker/analyzer/aot (They are going to share tests soon, so we will need to do it in all 3).

Reply:
I wonder if it would be better to just standardize on however reflection displays these. This is the same issue as with interpreting ref/in/out in method signatures. C# tends to come up with new things every release. Do we want to be in a position where we need to play catch up, or in the position of reflection that just doesn't care?

https://github.com/dotnet/linker/issues/2158#issuecomment-982181457 (ref/in/out comment here)

Reply:
I must admit I thought about it - but I was lazy to fix probably 100s of places in the linker tests to change this.
Another way would be to implement some smarts to the test validation system to let it allow a difference here - probably doable - didn't try.